test-lib.sh

```diff
@@ -4,24 +4,46 @@
 #
 # Always use sourced from a specific container testfile
 #
-# reguires definition of CID_FILE_DIR
-# CID_FILE_DIR=$(mktemp --suffix=<container>_test_cidfiles -d)
-# reguires definition of TEST_LIST
-# TEST_LIST="\
-# ctest_container_creation
-# ctest_doc_content"
 
 # Container CI tests
 # abbreviated as "ct"
 
+# run ct_init before starting the actual testsuite
+
 # may be redefined in the specific container testfile
 EXPECTED_EXIT_CODE=0
-export TESTSUITE_RESULT=0
```

Comment:
Why not exported? It should be used either for OpenShift 4 tests, right?

Reply:
It is now in the init function.